src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/cache.html.twig

```diff
@@ -0,0 +1,120 @@
+{#
+ # This file is part of php-cache\cache-bundle package.
+ #
+ # (c) 2015-2015 Aaron Scherer <aequasi@gmail.com>
+ #
+ # This source file is subject to the MIT license that is bundled
+ # with this source code in the file LICENSE.
+ #}
+
+{% extends 'WebProfilerBundle:Profiler:layout.html.twig' %}
+
+{% block head %}
+    {{ parent() }}
+    <style>
+        .color-red {
```

Comment:
would be better to include necessary styles in the CSS, to be usable for all collectors